src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -1080,7 +1080,9 @@ public static RemoveContainerParam forceKill(final boolean force) {
    *                              if container is not found (404)
    * @throws DockerException      if a server error occurred (500)
    * @throws InterruptedException If the thread is interrupted
+   * @deprecated Replaced by {@link #archiveContainer(String, String)}
```

Comment:
Can we add a note that this was deprecated since 1.20 and will throw `UnsupportedApiVersionException` for 1.24 and above?